ansible/roles/sonic-common/tasks/sensors_check.yml

```diff
@@ -5,6 +5,10 @@
 - set_fact: 
     ansible_python_interpreter: "docker exec -i {{ pmon_ps.stdout }} python"
 
+- set_fact:
+    minigraph_hwsku: "ACS-MSN2700"
```

Comment:
Can we swap those two? Since "Mellanox-SN2700" is the new official SKU we'd like to set ACS-MSN2700 to become Mellanox-SN2700 instead.